Create ui-c8y-1021-22-51-context-help-missing-in-administrations-localization-section-after-migration-from-dtm.md (#3087)
SoftwareAG/c8y-docs: content/change-logs/platform-services/ui-c8y-1021-22-51-context-help-missing-in-administrations-localization-section-after-migration-from-dtm.md

---
date: ""

> title: Contextual help is available for the localizations view in administration

Reviewer: ```suggestion
title: Contextual help is again available for the localization view in the Administration application
```

> title: Contextual help is again available for the localization view in the Administration application

Reviewer: ```suggestion
title: Contextual help is available again for the localization view in the Administration application
```

> title: Context help missing in Administrations Localization section after migration from DTM (#8409) [GRAFT][release/y2025] (#8525)

Reviewer 2: ```suggestion
title: Contextual help is available for the localizations view in administration
```

product_area: Platform services
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-0UgqXH1Ys
    label: Administration
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-62628
version: 1021.22.51
---

> During the migration of the localization view to the administration application, the contextual help button was removed. It is now available again.

Reviewer: ```suggestion
After the migration of the localization view to the Administration application, the contextual help button was unintentionally removed. This issue has been fixed and the context help is available again.
```

> Context help missing in Administrations Localization section after migration from DTM (#8409) [GRAFT][release/y2025] (#8525)

Reviewer 2: ```suggestion
During the migration of the localization view to the administration application, the contextual help button was removed. It is now available again.
```
